templates/donors.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="donors">
+  <div class="donors_image">
+    {{{asset image}}}
+  </div>
+  <div class="donors_info">
+    <p class="donors_description">{{description}}</p>
+    <div class="donors_link_container">
+      <div class="donors_ctaPrimary_link">
+        {{{module ctaPrimary}}}
+      </div>
+      <div class="donors_ctaSecondary_link">
```

Comment:
Please make sure to include an aria-label property so to give more context to screen readers

Reply:
thanks, i corrected it!!